ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.15.yaml

```diff
@@ -317,6 +317,40 @@ tests:
       COMPUTE_NODE_TYPE: m6g.xlarge
       FEATURE_SET: TechPreviewNoUpgrade
     workflow: openshift-e2e-aws-serial
+- as: ocp-e2e-ibmcloud-ovn-multi-ppc64le
+  cron: 20 7,19 * * *
```

Comment:
```suggestion
  cron: 20 7,17 * * *
```

Reply:
Let's not use the same exact execution hour as the 4.14 job, let's use 2 hrs earlier